README.md

```diff
@@ -286,13 +288,22 @@ Proofs referenced by content address MUST be resolvable by the recipient, for in
 
 The following capabilities are REQUIRED to be implemented.
 
-## 4.1 All Owned Resources
+## 4.1 Superuser
+
+The superuser action MUST be denoted `*`. This is the maximum ability, and may be applied to any resource (it is the "top" ability).
+
+This is useful in several cases, for example:
+
+1. When delegating all resources, like in a [`my` scheme](#4.2-all-owned-resources)
+2. To grant the maximum ability when the current ability semantics may be extended later
+
+## 4.2 All Owned Resources
 
-### 4.1.1. `my` Scheme
+### 4.2.1. `my` and `as` Schemes
```

Comment:
This was in an earlier draft, but then I tried to cover it with `my` since they're pretty similar. It's much clearer this way, so I'm glad that Gozala spoke up!